otter/test/cloud_client/test_init.py

```diff
@@ -600,6 +602,94 @@ def test_perform_srvreq_nested(self):
              self.throttler, 1, ereq.intent))
 
 
+def assert_parses_common_clb_errors(testcase, intent, eff, lb_id="123456"):
```

Comment:
This function is factored out from `CLBClientTests` so that it can be used in newer `GetCLBNodeFeedTests`. It had much more calls from `CLBClientTests` which used "123456" as lb_id as compared to one call from `GetCLBNodeFeedTests` which used different lb_id. Hence, the default value. But this sounds like a stupid reason to have default value especially since this function is outside both the classes. I'll remove it. Thanks for catching this :)